netlify.toml

```diff
@@ -45,6 +47,7 @@
   VITE_SANITY_DATASET = "staging"
   VITE_RECAPTCHA_SITE_KEY = "6LchxlAaAAAAANjtO3w4b5ihVCL87tYfbe0ltzp9"
   VITE_API_URI = "https://api-staging.regen.network"
+  VITE_APP_URL="http://localhost:3000"
```

Comment:
why not just use `window.location.origin`?

Reply:
Fair point, I think it should be better indeed.

Reply:
yes we already use this in other places in the app so best to have it consistent and we can migrate everything to be ssr friendly when needed